app/controllers/search_controller.rb

```diff
@@ -0,0 +1,8 @@
+class SearchController < ApplicationController
+
+  def index
+    @start_address = params[:location]
+    @station = NrelFacade.nearest_station(@start_address)
+    @route = MapquestFacade.route(@start_address, @station.address)
```

Comment:
You could also choose to have one facade that calls the nrel_service and mapquest_service, but I don't have a strong preference. The advantage of doing it this way is that these facades are loosely coupled and therefore could be used for other functionality in the future.